st2auth_ldap_backend/ldap_backend.py

```diff
@@ -136,7 +142,9 @@ def _ldap_connect(self):
         """
         try:
             connection = ldap.initialize(self._ldap_uri)
-            ldap.set_option(ldap.OPT_PROTOCOL_VERSION, 3)
+            connection.set_option(ldap.OPT_PROTOCOL_VERSION, 3)
+            connection.set_option(ldap.OPT_REFERRALS,
+                                  int(self._chase_referrals))
```

Comment:
Looks like this is going to be in conflict w/ PR https://github.com/StackStorm/st2-auth-backend-ldap/pull/7.